packages/soundxyz/src/Soundxyz.ts

```diff
@@ -137,27 +137,26 @@ export const simulateMint = async (
 export const getProjectFees = async (
   mint: MintActionParams,
 ): Promise<bigint> => {
-  const { chainId, contractAddress } = mint
+  const { chainId, contractAddress, tokenId } = mint
 
+  const tier = tokenId ?? 0
   const client = createPublicClient({
     chain: chainIdToViemChain(chainId),
     transport: http(),
   })
 
-  const mintInfoList = (await client.readContract({
+  const nextSchedule = (await client.readContract({
     address: SUPERMINTER_V2,
     abi: MINT_INFO_LIST_ABI,
-    functionName: 'mintInfoList',
-    args: [contractAddress],
-  })) as MintInfoList
-
-  const mintInfo = mintInfoList[0]
+    functionName: 'nextScheduleNum',
+    args: [contractAddress, tier],
+  })) as bigint
```

Comment:
For some reason I think this is being cast as `number` instead of `bigint`

Lets try to use `number` and then cast into a `BigInt` below in the args